test/routes_helpers.js

```diff
@@ -14,3 +23,49 @@ exports.getRoute = function (routes, path) {
 
   return route
 }
+
+exports.makeAccountRoutes = function (options, requireMocks) {
```

Comment:
I just want to flag up that if we do break up `lib/routes/account.js` to match the changes here, it's unlikely that this will be a common function any more and each of the new route modules will need their own `makeRoutes` implementation.

So this refactoring might be premature (or not, if we're not going to break up the production code).